src/main/java/bio/terra/service/filedata/azure/tables/TableServiceClientUtils.java

```diff
@@ -0,0 +1,35 @@
+package bio.terra.service.filedata.azure.tables;
+
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.data.tables.TableClient;
+import com.azure.data.tables.TableServiceClient;
+import com.azure.data.tables.models.ListEntitiesOptions;
+import com.azure.data.tables.models.TableEntity;
+
+public class TableServiceClientUtils {
```

Comment:
@giventocode  - I would definitely appreciate a review here! Maybe this is something that could live in CRL.

Reply:
Sorry for the delayed response. I missed this notification. Left some comments below.

Reply:
No worries! Thanks for the review!